lib/open-search-client.js

```diff
@@ -195,6 +195,9 @@ const createTermsIndex = (prefixUrl, index) => createIndex(prefixUrl, index, {
       source_terminology: {
         type: 'keyword',
       },
+      graph_uri: {
```

Comment:
Could we camelCase this? I'd like to fix `source_terminology` too if you want to do that in this PR.

Reply:
Fixed in 84b7185